### DISCUSSION TITLE
Behaviour when selecting images

### DISCUSSION THREAD

**Discussion by aberenguel** [sepinf-inc/IPED#1084] [General]
When I show the gallery tab to select images, the selected image is shown in the ImageViewer panel.
If I press CTRL key, the last image of the selected set is shown. 
I expected that the last clicked image would be shown in the ImageViewer, not the last in the selected set.

**Comment by lfcnassif** [comment]
I was able to reproduce what you described in both 3.18.14 and 4.0.0-snapshot. I don't remember if a specific behavior was intended, yours or current one, but I agree your expectation is better. I'll open an issue to change it, thanks.

**Comment by lfcnassif** [comment]
Done: #1085
